integrations/bbot/app.py

```diff
@@ -0,0 +1,101 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+import json
+import logging
+from pathlib import Path
+
+from bbot.errors import BBOTError
+from bbot.scanner import Scanner
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+
+app = FastAPI()
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+class ScanRequest(BaseModel):
+    target: str
+    presets: list[str] = ["web-basic"]
+    modules: list[str] = []
+
+
+def get_output_json(scan_name):
+    """Read and parse output.json which is a NDJSON file into a structured list of events."""
+    json_path = Path(f"/opt/deploy/bbot/.bbot/scans/{scan_name}/output.json")
+    if not json_path.exists():
+        logger.warning(f"output.json not found at {json_path}")
+        return []
+
+    events = []
+    try:
+        with open(json_path, "r", encoding="utf-8") as file:
+            for line in file:
+                if line.strip():

```

Comment:
Here you call `strip()` but never change the value of `line`. Is this intended behaviour ?

Reply:
Meaning that the `line` value you're passing to `json.loads` is not stripped

Reply:
actually I am iterating here on each line one by one (for line in file) and if the line is not empty (if line.strip():) then only I am adding it to json.loads.

Reply:
Yes the if clause is good. 
Just wanted to point out that the `strip()` method does not change the original string.
So, if you call the method but not reassign the value it won't be stripped when you call `json.loads`. Is this intended behaviour?
E.g.
```python
>>> a = " a "
>>> a.strip()
'a'
>>> a
' a '
>>>
```
I think that a better solution would be:
```suggestion
                if line := line.strip():
```
so that when the not empty string is passed to the later function is actually stripped.

Reply:
Sure